README.md

```diff
@@ -124,7 +127,7 @@ spec("some feature") {
     }
 
     after() {
-        printf("%i tests run. %i failed.\n", test_count, tests_failed);
+        printf("%i tests run.  %i failed.\n", test_count, tests_failed);
```

Comment:
Are two spaces intentional here?

Reply:
I'm a "two spaces between sentences" kind of person.  If you're not so much, that's fine.  I don't recall whether I did that automatically or intentionally, off the top of my head, though.

Reply:
This is not really common I would say and I'm just afraid that I would get another PR from someone else putting it back to one space :) If you don't mind I would stick to defaults.